### ISSUE TITLE
XCUITests: Update tests with the new onboarding screen (autofill)

### ISSUE THREAD

**Issue by isabelrios** [mozilla-lockbox/lockbox-ios#689]
Once Autofill lands and the onboarding as part of it, tests need to be updated to take into account this new screen, they will fail until then:

![captura de pantalla 2018-09-07 a las 18 51 13](https://user-images.githubusercontent.com/1897507/45233097-f69a0c80-b2d1-11e8-81cd-6fb71374d8d5.png)

After tapping on Get Started, this is presented and depending on the option selected we would need to test the expected behavior.

**Comment by devinreams** [comment]
For now I have #692 just look for the "Set Up AutoFill" button and then tap the "Not Now" to get past and to the Welcome screen...
